.gitignore

```diff
@@ -7,3 +7,5 @@ venv/
 .pytest_cache/
 **__pycache__/
 **db.sqlite3
+.db3
```

Comment:
The `.db3` file extension is included, which is fine if you are using such database files. Ensure that this is intentional and that you actually use `.db3` files in your project.